ecommerce-examples/example1/back-end/javascript/new-checkout.js

```diff
@@ -0,0 +1,26 @@
+const chargebee = require('chargebee');
+module.exports = (req, res) => {
+  try {
+    const isPlan = req.query.item_type === 'plan';
+    const checkoutCallback = isPlan
+      ? chargebee.hosted_page.checkout_new_for_items
+      : chargebee.hosted_page.checkout_one_time_for_items;
+    delete req.query.item_type;
+    const params = isPlan
+      ? req.query
+      : {
+          ...req.query,
+          item_prices: req.query.subscription_items
```

Comment:
`item_prices` are used for `checkout_one_time_for_items` [api ref](https://apidocs.chargebee.com/docs/api/hosted_pages?prod_cat_ver=2#checkout_charge-items_and_one-time_charges
)
I believe it should be `subscription_items` for `checkout_new_for_items` [api_ref](https://apidocs.chargebee.com/docs/api/hosted_pages?prod_cat_ver=2#create_checkout_for_a_new_subscription)

Can you please confirm this works for both checkout-one-time and checkout-new cases?

Reply:
I see now what you're doing on the client side, but I think it would make sense to keep it clean and more readable, provided that this is going to be an example for devs.